tasks.php

```diff
@@ -580,7 +582,21 @@ function create_task_from_form_submission($formsub)
             TaskHeader("All Open Tasks");
             list_all_open_tasks();
             break;
-        }
+
+        case 'notify_new':
+            $userSettings =& Settings::get_Settings($pguser);
+            $userSettings->add_value('taskctr_notice', 'notify_new');
+            metarefresh(0, "$tasks_url");
+//            TaskHeader("All Open Tasks");
```

Comment:
Please remove commented-out code.

Reply:
doh! ... forgot to save file from editor.